portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+function getUserName(){
+    const userName =  fetch("/data").then(response => response.text()).then(name);
```

Comment:
+1 to what Casey said in a review for a different commit, I don't think you need this last .then(name) since it's not doing anything with the input

Reply:
> +1 to what Casey said in a review for a different commit, I don't think you need this last .then(name) since it's not doing anything with the input

Thank you Vita for catching this error. Indeed that last then is unnecessary. I will correct it asap.